src/components/Footer.tsx

```diff
@@ -37,6 +39,8 @@ export class Footer extends React.PureComponent<IFooterProps, IFooterState> {
     isSessionBusy: false,
     showSolutionButton: false,
     showRunButton: false,
+    utmSource: "datacamp_light",
```

Comment:
I thought it makes the most sense to defined the default values here where it is the closeste to where it's going to be used.